root/etc/cont-init.d/98-cloudflare

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/with-contenv bash
```

Comment:
Can you rename this file to be the same as the mod name, `98-cloudflare-real-ip` ?

Reply:
Done